desktop/src/main/java/bisq/desktop/main/offer/MutableOfferViewModel.java

```diff
@@ -1226,11 +1226,16 @@ private void maybeShowMakeOfferToUnsignedAccountWarning() {
         if (dataModel.getDirection() == OfferPayload.Direction.SELL &&
                 PaymentMethod.hasChargebackRisk(dataModel.getPaymentAccount().getPaymentMethod(), dataModel.getTradeCurrency().getCode())) {
             Coin checkAmount = dataModel.getMinAmount().get() == null ? dataModel.getAmount().get() : dataModel.getMinAmount().get();
-            if (checkAmount != null && !checkAmount.isGreaterThan(OfferRestrictions.TOLERATED_SMALL_TRADE_AMOUNT)) {
-                GUIUtil.showMakeOfferToUnsignedAccountWarning(btcFormatter);
+            Coin effectiveLimit = getEffectiveLimit();
+            if (checkAmount != null && !checkAmount.isGreaterThan(effectiveLimit)) {
+                GUIUtil.showMakeOfferToUnsignedAccountWarning(btcFormatter, effectiveLimit);
             }
         }
     }
+    
+    private Coin getEffectiveLimit() {
+      return btcValidator.getMaxValue() == null ? btcValidator.getMaxTradeLimit() : btcValidator.getMaxValue();
```

Comment:
would you mind explaining this change?

Reply:
The max trade limit is configured to be 1 XMR but this amount may be further reduced if the account isn't signed, etc.

This method provides the effective limit based on configuration and any deductions.  It's used to cap the input amount and display the correct limit in the popup warning.

Reply:
sorry I didn't notice it affected two different kinds of messages, added more comments/questions